packages/fx-core/src/plugins/resource/bot/azureOps.ts

```diff
@@ -199,10 +200,12 @@ export class AzureOperations {
     try {
       res = await axios.post(zipDeployEndpoint, zipBuffer, config);
     } catch (e) {
+      Logger.error(e.message);
       throw new ZipDeployError(e);
     }
 
     if (!res || !utils.isHttpCodeOkOrCreated(res.status)) {
+      Logger.error(JSON.stringify(res));
```

Comment:
Nit: do we need all the res? shall we just log the error code and message?

Reply:
There is no message in res, only status code.
![image](https://user-images.githubusercontent.com/72963627/123398322-f465e180-d5d5-11eb-858b-091302544b1e.png)

Reply:
Agree that we don't need to print headers and request which are not very helpful for trouble shooting. I think the error message should be in data field. Can we trigger some failure on this zip deploy step (or just ask bug creator how to repro this) and check what's the value of data when status code is not 200?

In general, status code and data is enough to be print

Reply:
Agree that we don't need to print headers and request which are not very helpful for trouble shooting. I think the error message should be in data field. Can we trigger some failure on this zip deploy step (or just ask bug creator how to repro this) and check what's the value of data when status code is not 200?

In general, status code and data is enough to be print